code/game/machinery/vending/vendor_types/squad_prep/squad_engineer.dm

```diff
@@ -25,6 +27,7 @@ GLOBAL_LIST_INIT(cm_vending_gear_engi, list(
 		list("ES-11 Mobile Fuel Canister", 4, /obj/item/tool/weldpack/minitank, null, VENDOR_ITEM_REGULAR),
 
 		list("EXPLOSIVES", 0, null, null, null),
+		list("M5510 Laser-Guided Rocket", 8, /obj/item/ammo_magazine/rocket/brute, null, VENDOR_ITEM_RECOMMENDED),
```

Comment:
Same as above req comment, this isn't necessarily obvious as a BRUTE rocket from the name alone.